geneowak/-StackOverflow-lite
# Ui templates

#### What does this PR do?
The pull request merges the UI Templates for the project into the main branch

#### Description of Task to be completed?
It has pages where a user can do the following:
i. View a list of recently asked questions on the platform
ii. View a question with all the answers posted for it and add an answer.
iii. Post a question.
It also has a profile page where a user can view:
i. The number of questions asked he has asked.
ii. The number of answers given he has given.
iii. The list of questions asked by the user with the most answers.
iv. The list of recent questions asked by the user.

#### How should this be manually tested?
Navigate to https://geneowak.github.io/-StackOverflow-lite and try navigating through all the pages to confirm that everything was captured.
